ova/Libraries/elastic_functions.sh

```diff
@@ -88,12 +88,12 @@ configure_kibana(){
 
 install_kibana_app(){
 
-    if [ "${STATUS_PACKAGES}" = "stable" ]; then
+    if [ "${PACKAGES_REPOSITORY}" = "prod" ]; then
         #Wazuh-app production repository
         sudo -u kibana ${usr_kibana}/bin/kibana-plugin install https://packages.wazuh.com/wazuhapp/wazuhapp-${WAZUH_VERSION}_${ELK_VERSION}.zip
     fi
 
-    if [ "${STATUS_PACKAGES}" = "unstable" ]; then
+    if [ "${PACKAGES_REPOSITORY}" = "dev" ]; then
```

Comment:
This URL will cause errors.

Reply:
Done in https://github.com/wazuh/wazuh-packages/pull/462/commits/3e33cfd6fe2cfa31bc0e7627ba1db39146b5a746